src/mantine-core/src/Select/Select.tsx

```diff
@@ -97,6 +97,7 @@ export function Select({
         rightSectionProps={{ style: { pointerEvents: 'none' } }}
         required={required}
         themeOverride={themeOverride}
+        defaultValue={placeholder ? 'xx-placeholder-xx' : null}
```

Comment:
We still want to support defaultValue from props, by setting it to null we do not provide this option

Reply:
Oh yeah good point.
Now if the user has both, what do you think should take preference - defaultValue from props, or the placeholder?

Reply:
I guess it should be something like `defaultValue || (placeholder ? 'xx-placeholder-xx'  : null)`